imagej/imagej.py

```diff
@@ -529,7 +545,7 @@ def _dataset_to_xarray(self, dataset):
             :return: xarray with reversed (C-style) dims and coords as labeled by the dataset
             """
             attrs = self._ij.py.from_java(dataset.getProperties())
-            axes = [(cast('net.imagej.axis.CalibratedAxis', dataset.axis(idx)))
+            axes = [(JObject(dataset.axis(idx), JClass('net.imagej.axis.CalibratedAxis')))
                     for idx in range(dataset.numDimensions())]
```

Comment:
If this is used in a loop it is better to import the Java class once rather than using JClass as JClass requires a name lookup on the Java side rather than simply checking the local Python dictionaries.   As this is in a list comprehension I would recommend the lookup first style.

Also the cast operator `(type)@` may be used in place of the JObject.   Though that is a matter of preference in terms of style.

ie.
```
   from net.imagej.axis import CalibratedAxis
   axes = [(CalibratedAxis)@dataset.axis(idx) for idx in range(dataset.numDimensions())]
```